docs/snippets/react/nextjs-add-framework.ts.mdx

```diff
@@ -0,0 +1,20 @@
+```ts
+// .storybook/main.ts
+import { StorybookConfig } from '@storybook/nextjs';
+
+const config: StorybookConfig = {
+  // ...
+  framework: {
+    // name: '@storybook/react-webpack5', // 👈 Remove this
+    name: '@storybook/nextjs', // 👈 Add this
+    options: {
+      builder: {
+        // Set useSWC to true if you want to try out the experimental SWC compiler in Next.js >= 14.0.0
+        useSWC: true,
+      },
+    },
```

Comment:
`useSWC` does not exist in Storybook 8.0 anymore. See [the migration doc](https://github.com/storybookjs/storybook/blob/valentin/remove-babel-from-webpack5-builder/MIGRATION.md#automatic-swc-mode-detection).